### DISCUSSION TITLE
Adjusting quantile levels of Chronos Bolt model

### DISCUSSION THREAD

**Discussion by Silas2701** [amazon-science/chronos-forecasting#215] [Ideas]
As of now, I don't see a way to alter the quantile levels for Chronos Bolt when forecasting. Have you considered to implement this in the future or is there a way to actually change it because I tried several ways to adjust the config and none of them picked up the changes.
It seems a little restrictive to only be able to forecast the default `quantiles = [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9]`, respectively  not being able to predict quantiles below 0.1 quantile and beyond 0.9 quantile. I would also like to discover minimum and maximum values of the forecasts.

**Comment by abdulfatir** [comment]
@Silas2701 Since Chronos-Bolt models have been trained with `quantiles = [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9]`, if you select any other values, it will only be an interpolation or extrapolation. May I know what kind of use case you're working with for which you need higher (>0.9) and lower (<0.1) quantiles?

**Reply by Silas2701** [reply]
@abdulfatir I see, thank you for your response! I was just interested in forecasting prediction intervals with Chronos Bolt apart from the 80% prediction interval, e.g. 95% or even 99%. I wanted to use them to compare against the forecasts and intervals produced by other models.

**Reply by abdulfatir** [reply]
@Silas2701 Chronos-2 supports a wider quantile grid from 0.01 to 0.99. Hope this helps.

**Comment by doolingdavidrs21** [comment]
I have some issue; having extreme quantiles then allows for some natural anomaly detection techniques.

**Reply by abdulfatir** [reply]
@doolingdavidrs21 This requirement is on our radar for the next versions of the model. For now, you can probably make some assumptions downstream an obtain extreme quantiles. For example, you can fit a Student's-t distribution to the output quantiles per time-step and get extreme questions.

**Reply by abdulfatir** [reply]
@doolingdavidrs21 Chronos-2 supports a wider quantile grid from 0.01 to 0.99. Hope this helps.
